packages/eslint-config-eventbrite-react/rules/react-a11y.js

```diff
@@ -4,6 +4,55 @@ module.exports = {
     // View link below for docs on react a11y rules
     // https://github.com/evcohen/eslint-plugin-jsx-a11y/
     rules: {
+
+        // Enforce emojis are wrapped in and provide screenreader access.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md
+        'jsx-a11y/accessible-emoji': 'error',
+
+        // Enforce all anchors to contain accessible content.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-has-content.md
+        'jsx-a11y/anchor-has-content': 'error',
+
+        // Enforce a clickable non-interactive element has at least one keyboard event listener.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md
+        'jsx-a11y/click-events-have-key-events': 'error',
```

Comment:
do you know what this does for naturally clickable tags like `<button>` and `<a>`. They don't need the keypresses because the browser does it by default